7.1-alpine/entrypoint

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 set -e
 
+if [ "$ENABLE_XDEBUG" == "true" ]; then
+    docker-php-ext-enable xdebug
```

Comment:
this will output onto stdout when running the tests? 

I would suggest:

1. We add a >> /dev/null
2. I guess the -e above already handles it, but in case $? is different than 0, exit and print error about not being able to enable xdebug.